proxylib/proxylib/types.go

```diff
@@ -21,12 +21,8 @@ import "C"
 
 // Mirror C types to be able to use them in other Go files and tests.
 
-type OpType uint32
-type OpError uint32
-type Op struct {
-	op      uint32
-	n_bytes uint32 // The number of bytes of data the operation 'op' applies to.
-}
+type OpType int64
```

Comment:
Why aren't these `uint64`?

Reply:
C types cannot be used in multiple packages (here `main` and `proxylib`; see my responses to Ray's questions for more context), so I had to change the Go-representation of `FilterOp` type to an array instead of a struct. Go length types are (64-bit) signed, so I went for int64 for the array type.